fastlane/Fastfile

```diff
@@ -59,6 +59,58 @@ lane :github_release do |options|
   )
 end
 
+desc "Update dependencies to latest GitHub releases and opens a PR"
+lane :open_pr_upgrading_dependencies do |options|
+  if options[:dry_run]
+    dry_run = true
+  end
+  
+  new_versions = update_native_dependencies_to_latest()
+  
+  repo_status = Actions.sh("git status --porcelain")
+
+  repo_clean = repo_status.empty?
+
+  if repo_clean
+    UI.success('Git status is clean, nothing to update! 💪')
+    next
+  end
+
+  new_ios_version = new_versions[:new_ios_version]
+  new_android_version = new_versions[:new_android_version]
+
+  branch = "update-#{new_ios_version}-#{new_android_version}"
+  
+  result = Actions.sh("git checkout -b #{branch}")
+  if result =~ /error:(.)/
+    UI.error("Git couldn't change to `#{branch}`. Reason: #{result}.")
+  else
+    UI.success("Git switched to branch `#{branch}, all good! 💪")
+  end
+  
+  result = Actions.sh("git commit -am \"Upgrades versions\"")
+  if result =~ /error:(.)/
+    UI.error("Git couldn't create commit. Reason: #{result}.")
+  end
+
+  if dry_run
+    UI.message("ℹ️  Nothing more to do, dry_run: true")
+  else
+    push_to_git_remote(
+      remote: "origin",
+      set_upstream: true
+    )
+    create_pull_request(
+      api_token: ENV["GITHUB_TOKEN"],
```

Comment:
This has to be explicitly set otherwise it doesn't work, I think it might be trying to get another ENV variable